app/Livewire/Project/New/GithubPrivateRepositoryDeployKey.php

```
@@ -133,6 +140,31 @@ public function setPrivateKey($private_key_id)
+
+            $detector = new RepositoryDetector(
+                repositoryUrl: $this->repository_url,
+                branch: $this->branch ?? 'main',
+                baseDirectory: $this->base_directory ?? '/',
+                serverId: (int) $serverId,
+                teamId: $teamId,
+            );
+
+            $this->applyDetectionResult($detector->detect());
+        } catch (\Throwable $e) {
+            Log::debug('Repository detection failed in component', ['error' => $e->getMessage()]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Log' in method 'detectRepository'. (undefined)

(StaticAccess)